homeassistant/components/file/sensor.py

```diff
@@ -75,6 +89,7 @@ def __init__(
         self._file_path = file_path
         self._attr_native_unit_of_measurement = unit_of_measurement
         self._val_tpl = value_template
+        self._attr_unique_id = f"sensor_{file_path}"
```

Comment:
```suggestion
        self._attr_unique_id = slugify(file_path)
```

Reply:
Updated, the file name is not unique if multiple sensors are configured.